beeload/action.yaml

```diff
@@ -0,0 +1,116 @@
+name: 'beeload'
```

Comment:
Originally Beeload was meant as a general toolkit for working with Swarm in Github Actions with having additional features of the PR Previews, but originally the main goal was uploading data to Swarm. Since now you have the "unit action" `upload-dir` and others I would argue that the only feature of Beeload left here is the PR previews themselves. I would therefore suggest focusing this action only on PR preview and therefore renaming this action into something like `swarm-previews` or similar and removing the `preview` input (as it will be default `true`).

Reply:
Sure, what about `pr-previews`? `swarm` is already in the `swarm-actions` name, so seems a bit redundant.